ansible/tasks/installOpenwhiskCatalog.yml

```diff
@@ -5,6 +5,7 @@
     catalog_location={{ item.value.location }}
     catalog_repo_url={{ item.value.url }}
     api_host={{ groups['edge'] | first }}
```

Comment:
Will that work when there is a router and hence API host is the intended target not edge host? E.g. Blue green scenario.